examples/common-switches.mk

```diff
@@ -60,6 +60,10 @@ TIME_SYNC           ?= 0
 UDP_FORWARD         ?= 0
 
 
+ifeq ($(ACK_FP),1)
```

Comment:
This it is platform specific. However, it is now controlling the core behavior: whether to verify the `mAckedWithFramePending` flag. This switch is temporary, after all platforms implemented setting `mAckedWithFramePending` correctly, this will be removed.

Reply:
Rather than having a build option, how about updating the examples by always setting `mAckedWithFramePending = true` (with a comment, of course) if the correct way to do it is not immediately obvious?